bucket/fvie.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.5.0.0",
+    "description": "File VersionInfo Editor - A portable tool to edit VersionInfo metadata from Windows executables. Supports edition of EXE, DLL, and SYS files.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.carifred.com/License.php"
+    },
+    "homepage": "https://www.carifred.com/fvie/",
+    "url": "https://www.carifred.com/fvie/FVIE.exe",
+    "hash": "cf4d0442b0a324358eb0b420c34a62ee41e1a39339e2842ed92cf5eee1aa9641",
+    "bin": "FVIE.exe",
+    "shortcuts": [
+        [
+            "FVIE.exe",
+            "FVIE"
+        ]
+    ],
+    "checkver": {
```

Comment:
Same as homepage

```suggestion
    "checkver": "Current version:\\s+([\\d.]+)",
```

Reply:
So the `url` parameter of the `autoupdate` field defaults to the `homepage`? Ok, didn't know of that!

Reply:
No. `url` parameter of **`checkver`** is considered homepage when no other is specified.